### ISSUE TITLE
Any chance we can have an option to disable this for admins...

### ISSUE THREAD

**Issue by Sophist-UK** [JonathanMH/Append-Link-on-Copy#1]
... or logged on users?

**Comment by JonathanMH** [comment]
good point, I will have a look. In the meantime, feel free to submit a pull request

**Comment by Sophist-UK** [comment]
I will if I have the time, but at the moment I am concentrating on a Joomla site I run and don't have much time for the WP site I use this on.

But a quick look at the code suggests that it may not be too difficult to achieve.
